tomato-timer/public/index.html

```diff
@@ -1,19 +1,6 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <script src="https://www.gstatic.com/firebasejs/4.1.1/firebase.js"></script>
-      <script>
-        // Initialize Firebase
-        var config = {
-          apiKey: "AIzaSyCZytKSL4snOizTFVq_lNyA3bRKHUqrvyw",
-          authDomain: "tomato-timer-30220.firebaseapp.com",
-          databaseURL: "https://tomato-timer-30220.firebaseio.com",
-          projectId: "tomato-timer",
-          storageBucket: "tomato-timer.appspot.com",
-          messagingSenderId: "37256474259"
-        };
-        firebase.initializeApp(config);
-      </script>
```

Comment:
This is why PRs are a great thing! Your peers will catch stuff you have long since forgotten you didn't even use anymore.